TFNetworkLayer.py

```diff
@@ -6189,6 +6192,12 @@ def py_write(data_np, tags, *sizes):
         n_batch = data_np.shape[0]
         assert len(sizes) == len(data.size_placeholder)
         seq_lens = {i: size for (i, size) in zip(sorted(data.size_placeholder.keys()), sizes)}
+        # there may be axes with a fixed length other than the batch and feature axes
+        # these have the indices 0, ..., (ndim-1), as the batch dimension is skipped
+        for dim in range(ndim-1):
```

Comment:
Please use PyCharm and check for Python formatting warnings.

Reply:
PyCharm didn't warn me about any issues - I've capitalized the beginning of the sentences and added periods. Is that what you're referring to? (Sorry for the late reply, I mistakenly never submitted my comment)

Reply:
`ndim-1`. There should be spaces before and after the operand `-`.